packages/venia-concept/src/actions/checkout/asyncActions.js

```diff
@@ -197,7 +201,9 @@ export function formatAddress(address = {}, countries = []) {
     const region = regions.find(({ code }) => code === region_code);
 
     if (!region) {
-        throw new Error(`Region "${region_code}" is not an available region.`);
+        throw new Error(
+            `State "${region_code}" is not an valid state abbreviation.`
```

Comment:
Oh wait, are `state` and `region` the same thing?

Reply:
james's asked me to rename region to state, since venia is US-oriented store